athenahealth/insurance_test.go

```diff
@@ -201,3 +202,30 @@ func TestHTTPClient_UploadPatientInsuranceCardImage(t *testing.T) {
 
 	assert.True(result.Success)
 }
+
+func TestHTTPClient_GetPatientInsurancePackage(t *testing.T) {
+	assert := assert.New(t)
+
+	called := false
+	h := func(w http.ResponseWriter, r *http.Request) {
+		called = true
```

Comment:
Can we also assert the method and path?

```suggestion
		called = true
		assert.Equal(http.MethodGet, r.Method)
		assert.Equal("/patients/1/insurances/2/image", r.URL.String())
```